action.yml

```diff
@@ -1,15 +1,16 @@
-name: Notion Issue Sync
-description: Invokes a provided Make web-hook for syncing the state of an issue linked to the active PR
+name: 'notion-sync-issue'
+description: 'Provide a description here'
+author: 'subhroacharjee'
```

Comment:
What happened here? No need to change any of this metadata. Please revert to original